lib/node/runner/checker.go

```diff
@@ -465,6 +465,7 @@ func INITBallotValidateTransactions(c common.Checker, args ...interface{}) (err
 		Ballot:         checker.Ballot,
 		Transactions:   checker.Ballot.Transactions(),
 		VotingHole:     voting.NOTYET,
+		transactions:   map[string]transaction.Transaction{},
```

Comment:
There are two similar fields `Transactions` and `transactions` in `BallotTransactionChecker`.
`Transactions` is []string type and has hashes of transactions.
`transactions` is map[string]Transaction type and has transactions and hashes.
I don't know exactly but can we use only one field `transactions map[string]Transaction`?

Reply:
@Charleslee522 `Transaction` is created by `Balot.Transactions()`, without real message. `transaction` will contain the real transaction message. How about renaming to `transactionCache`?

Reply:
@spikeekips Alright, the `transactionCache` is good.